src/main/java/harmonised/pmmo/features/mobscaling/MobAttributeHandler.java

```diff
@@ -85,6 +88,31 @@ private static void handle(LivingEntity entity, ServerLevel level, Vec3 spawnPos
 				.map(ResourceLocation::new)
 				.collect(Collectors.toSet());
 
+		for (harmonised.pmmo.config.codecs.AttributeModifier att : dimModsCustom) {
+			var attributeLocation = new ResourceLocation(att.attribute());
+			var attribute = ForgeRegistries.ATTRIBUTES.getValue(attributeLocation);
+			if (attribute == null) continue;
+
+			AttributeInstance attributeInstance = entity.getAttribute(attribute);
+			if (attributeInstance == null) continue;
+
+			double bonus = att.amount();
+			AttributeModifier.Operation operation = switch (att.operation()) {
+				case "addition" -> AttributeModifier.Operation.ADDITION;
+				case "multiply_base" -> AttributeModifier.Operation.MULTIPLY_BASE;
+				case "multiply_total" -> AttributeModifier.Operation.MULTIPLY_TOTAL;
+				default -> AttributeModifier.Operation.ADDITION;
+			};
+			UUID modifierID = switch (att.operation()) {
+				case "addition" -> CUSTOM_MOD_ID_ADDITION;
+				case "multiply_base" -> CUSTOM_MOD_ID_MULTIPLY_BASE;
+				case "multiply_total" -> CUSTOM_MOD_ID_MULTIPLY_TOTAL;
```

Comment:
The use of the `Operation` enum in `MobModifier` also makes this a cleaner and more efficient switch statement and removes the need for a default case.